plugins/azure-sites/src/components/AzureSitesOverviewTableComponent/AzureSitesOverviewTable.tsx

```diff
@@ -99,15 +99,17 @@ const Kind = ({ value }: { value: Kinds }) => {
 };
 
 const Tags = ({ tags }: { tags: any }) => {
-  return Object.keys(tags).map((key: any) => (
-    <Chip
-      key={key}
-      label={`${key}: ${tags[key]}`}
-      size="small"
-      variant="default"
-      style={{ marginBottom: '1px' }}
-    />
-  ));
+  return Object.keys(tags)
+    .sort()
+    .map((key: any) => (
+      <Chip
+        key={key}
+        label={`${key}: ${tags[key]}`}
+        size="small"
+        variant="default"
+        style={{ marginBottom: '1px' }}
+      />
+    ));
 };
```

Comment:
Instead of pre-sorting the column you can leave this to the Table & provide this as a default, so smth. like this:

https://github.com/backstage/backstage/blob/f750e51e78d836bd3b4ac50692b7c92627ae86b9/plugins/azure-devops/src/components/GitTagTable/GitTagTable.tsx#L33-L42

You could even consider implementing a `customSort`. 

I do not know much about `AzureSites` - can you clarify why you choose the `Tag` instead of e.g. `lastModified`, which is also available?

Reply:
These are the multiple tags displayed in a single cell in the table, not in a column with one tag per row. This sorts the tags within the cell so they don't jump around on each refresh. I assume the sort on the table for when you sort the rows by the value in that column ? (I'd refrained from adding an image to the original description as almost everything in there is sensitive to a degree, however heres one with a rediculous amount of editing to show what is displayed for the tags)

![image](https://github.com/backstage/backstage/assets/1361894/182949c5-de69-434d-86cf-166cc53fcb1c)

Reply:
(I have a separate PR in the backend component to filter out those hidden-* tags that should not be shown in the UI)

Reply:
@tudi2d re: 

> I do not know much about AzureSites - can you clarify why you choose the Tag instead of e.g. lastModified, which is also available?

This is the order the tag 'lozenges' are displayed in the cell (see above screen shot), not the order of the rows when ordered by that column.

The order tags are sorted on is not particularly critical, then main thing is that it is consistent on each render. Currently, due to the fact that javascript objects don't have a determinate order of properties the tags will have a different order on every fetch of the data. This causes the tags to "jump around" when you refreshteh data from the backend (every 15 seconds). This wasn't as noticible previously, when the table would reload and show the loading animation every 15 seconds (which is what this PR is trying to address, it is incredibly annoying to have your table elements disappear for a few seconds every 15 seconds while you are trying to interact with them).

I'd be just as happy if there was no refesh at all, but as this appears to be as-designed, I'd at least like to fix the UI to not disappear.

Reply:
ah - thanks for the clarification & the screenshot. forgot how this was displayed. makes sense now, thanks!